sql/scalar-functions/time-and-date.mdx

```diff
@@ -0,0 +1,482 @@
+---
+title: Time and date functions
+description: Scalar functions for handling times and dates
+---
+
+{/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+ */}
+
+_Arroyo's Scalar function implementations are based on
+[Apache DataFusion](https://arrow.apache.org/datafusion/) and these docs are derived from
+the DataFusion function reference._
+
+
+## `now`
+
+Returns the current UTC timestamp.
+
+The `now()` return value is determined at query time and will return the same timestamp,
```

Comment:
We should error on these time functions, probably.

Reply:
Agreed; removed them from the docs